selfdrive/car/chrysler/values.py

```diff
@@ -41,7 +41,7 @@ class CAR(Platforms):
     CHRYSLER_PACIFICA_2017_HYBRID.specs,
   )
   CHRYSLER_PACIFICA_2019_HYBRID = ChryslerPlatformConfig(
-    [ChryslerCarDocs("Chrysler Pacifica Hybrid 2019-23")],
+    [ChryslerCarDocs("Chrysler Pacifica Hybrid 2019-24")],
```

Comment:
it seems like in 2024 they just switched to using `Plug-in Hybrid` in their brochure, but it's always been a plug-in hybrid. Keeping it as-is to avoid confusion

https://cdn.dealereprocess.org/cdn/brochures/chrysler/2023-pacifica.pdf

https://www.chryslereguide.com/files/BuyersGuide/BuyersGuide_Pacifica.pdf